### DISCUSSION TITLE
FEATURE REQUEST: CARDANO APP SHOULD WORK IN APP NOT WITH OTHER WALLETS

### DISCUSSION THREAD

**Discussion by nickyrabit** [LedgerHQ/ledger-live-desktop#3653] [General]

**Comment by CallMeCom** [comment]
Ledger Live ADA support would be amazing, it is the only large-cap crypto that I hold that has support issues. I see smaller cap projects with far better Ledger integration.

**Comment by gre** [comment]
please refer to https://github.com/LedgerHQ/ledger-live-desktop/discussions/3420

**Reply by CallMeCom** [reply]
I went over there and see its just another user having the same issues we all are.  We are being told here that another service needs to add support to ledger live, then when we go ask them they come back and tell us that this is something only Ledger can do.... meanwhile we are all frustrated and when we open tickets they are closed or we are moved to another ticket that doesn't give us any closure or answers - which doesn't help with the frustration.... then we hear the leader of the Cardano project on a livestream say they want to get onto Ledger Live but are trying to find a way to make it affordable it sounds like they are being asked for a very serious sum to get listed which further frustrates anyone that PAID for these devices!

I have several devices, for myself and backups as well as for family that I have referred since I truly believed in your product, but the way this has been dealt with has really felt like you pushing us right out of the door and onto the street to find a new product which is what I am currently researching right now. This is too much for me to just want to keep track of my assets in one place.

**Reply by johnpc** [reply]
It's not true at all. Trust Wallet multicoin wallet did it and it's open source https://github.com/trustwallet/wallet-core. See discussion here https://github.com/LedgerHQ/ledger-live-desktop/discussions/4010
